packages/react-sdk/src/components/Reaction/Reaction.tsx

```diff
@@ -15,33 +13,32 @@ export const defaultEmojiReactions: Record<string, string> = {
   ':fireworks:': '🎉',
 };
 
-export const Reaction = (props: ReactionProps) => {
-  const { reaction, sessionId, call, hideAfterTimeoutInMs = 5500 } = props;
-  const [isShowing, setIsShowing] = useState(false);
+export const Reaction = ({
+  participant: { reaction, sessionId },
+  hideAfterTimeoutInMs = 5500,
+}: ReactionProps) => {
+  const call = useCall();
+
   useEffect(() => {
-    let timeoutId: NodeJS.Timeout;
-    if (reaction) {
-      setIsShowing(true);
-      timeoutId = setTimeout(() => {
-        setIsShowing(false);
-        call.resetReaction(sessionId);
-      }, hideAfterTimeoutInMs);
-    }
+    if (!call || !reaction) return;
+
+    const timeoutId = setTimeout(() => {
+      call.resetReaction(sessionId);
+    }, hideAfterTimeoutInMs);
+
     return () => {
       clearTimeout(timeoutId);
     };
   }, [call, hideAfterTimeoutInMs, reaction, sessionId]);
 
-  const { emoji_code } = reaction;
+  if (!reaction) return null;
+
+  const { emoji_code: emojiCode } = reaction;
+
   return (
-    <div
-      className={clsx(
-        'str-video__reaction',
-        isShowing && 'str-video__reaction--visible',
-      )}
-    >
+    <div className="str-video__reaction">
       <span className="str-video__reaction__emoji">
-        {emoji_code && defaultEmojiReactions[emoji_code]}
+        {emojiCode && defaultEmojiReactions[emojiCode]}
```

Comment:
Don't we want to make the emoji set customizable? the `default` part of the `defaultEmojiReactions` variable would imply that.